tools/delete-artifacts-folder

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# In case of a test failure and retry, we want to delete previously recorded
+# artifacts and coverage because it might cover different code paths in case of
+# success, and this leads to flaky coverage reports.
+
+set -e
+
+rm -vfr "cover_db$COVERDB_SUFFIX"
+[ -d "$OPENQA_FULLSTACK_TEMP_DIR" ] && rm -vfr "$OPENQA_FULLSTACK_TEMP_DIR"
```

Comment:
hm, this will delete `test-results/fullstack` completely, including result directories for previously running tests.
I don't have a simple idea how to prevent that right now. Maybe add `COVERDB_SUFFIX` to the variable. (And rename `COVERDB_SUFFIX` into something more general.)
Since we're currently only interested in the fullstack results, it might be ok though.

Reply:
It is still better than having nothing at all or broken retries due to the leftovers. However, one could likely simply move everything into a different directory (or a sub directory).

Reply:
Let's make the folder $target-specific then